analyser_merge_milestone_FR_metropole.py

```diff
@@ -0,0 +1,120 @@
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from collections import OrderedDict
+from .Analyser_Merge import Analyser_Merge, Source, CSV, Load, Mapping, Select, Generate
+
+class Analyser_Merge_Milestone_FR_metropole(Analyser_Merge):
+    def __init__(self, config, logger = None):
+        Analyser_Merge.__init__(self, config, logger)
+
+        # fix item 
+        self.def_class_missing_official(item = 8130, id = 41, level = 3, tags = ['merge', 'milestone'],
+            title = T_('Milestone not integrated'))
+        self.def_class_possible_merge(item = 8131, id = 43, level = 3, tags = ['merge', 'milestone'],
+            title = T_('Milestone integration suggestion'))
+        self.def_class_update_official(item = 8132, id = 44, level = 3, tags = ['merge', 'milestone'],
+            title = T_('Milestone update'))
+
+#            u"https://www.data.gouv.fr/fr/datasets/gestionnaires-du-reseau-routier-national/",
+#            u"Bornage du réseau routier national",
+#            CSV(Source(u"data.gouv.fr:Ministère de la Transition écologique et solidaire", millesime = "01/2018",
+#                    fileUrl = u"https://www.data.gouv.fr/fr/datasets/r/9045452c-f6a6-4e13-b82c-6f55e671656d",
+#                    filter = lambda text: text.replace(',', '.')), separator = u"\t"),
+# need to read this file in first : route + depPrD + prD is unique code match Gestionnaire => operator to second csv...?
+
+        self.init(
+            u"https://www.data.gouv.fr/fr/datasets/bornage-du-reseau-routier-national/",
+            u"Bornage du réseau routier national",
+            CSV(Source(u"data.gouv.fr:Ministère de la Transition écologique et solidaire", millesime = "01/2019",
+                    fileUrl = u"https://www.data.gouv.fr/fr/datasets/r/fbc8b73b-a65c-486b-a710-ed22b9e4070c",
+                    filter = lambda text: text.replace(',', '.')), separator = u"\t"),
+                    
+            Load("x", "y",srid = 2154,
+                where = lambda row: self.transform_to_natref(row) is not None),
```

Comment:
`where` is for condition. Is `self.transform_to_natref` a condition or tag mapping ? If is for tag mapping is sgould go in the mapping section.

Reply:
it's condition 
- if result is none 
and a tag for other value.